### DISCUSSION TITLE
TFTP: Dnsmasq (WHY?)

### DISCUSSION THREAD

**Discussion by keleticsaba** [antonym/netboot.xyz#1516] [Q&A]
At "Services/TFTP" i have: "TFTP: version 2.90 Copyright (c) 2000-2024 Simon Kelley" and not "thfp-hpa...", no mather how i pull docker image, and netbootxyz cannot get any image, but in the container console i can reach anything.
![Képernyőkép 2024-09-16 21-07-30](https://github.com/user-attachments/assets/e01c9bd9-4792-43d1-8fff-78c33fb27d25)

**Comment by centerionware** [comment]
Because dnsmasq can operate as a standalone tftp server.
